From db8ff1f783d8070860ae6e1704fbdce407805316 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Wed, 29 Jul 2020 06:33:17 +0300
Subject: [PATCH 004/116] Revert "FODCircleView: Reset background to Lineage
 state for default FOD icon"

This reverts commit a7be41fa8258669d3542911795d6b6d1d28b71ae.
---
 .../src/com/android/systemui/biometrics/FODCircleView.java     | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java b/packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java
index 3a25dfc613c..4baf495b560 100644
--- a/packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java
+++ b/packages/SystemUI/src/com/android/systemui/biometrics/FODCircleView.java
@@ -387,7 +387,8 @@ public class FODCircleView extends ImageView implements ConfigurationListener, T
 
     @Override
     protected void onDraw(Canvas canvas) {
-        if (!mIsCircleShowing && mSelectedIcon == 0) {
+        if (!mIsCircleShowing) {
+            mPaintFingerprintBackground.setXfermode(new PorterDuffXfermode(PorterDuff.Mode.SRC_IN));
             canvas.drawCircle(mSize / 2, mSize / 2, mSize / 2.0f, mPaintFingerprintBackground);
         }
         super.onDraw(canvas);
-- 
2.17.1

